tools/donate-cpu-server.py

```diff
@@ -272,8 +275,15 @@ def timeReport(resultPath):
                 text += filename[filename.find('/')+1:] + ' ' + splitline[2] + ' ' + splitline[1] + '\n'
```

Comment:
```suggestion
                 text += filename[filename.rfind('/')+1:] + ' ' + splitline[2] + ' ' + splitline[1] + '\n'
```

Reply:
Not sure: Is `find` used here instead of `rfind` by accident?

Reply:
yes I think so. I guess a rfind is more complex than necessary we could probably write 'len(resultPath) + 2' instead. feel free to use `rfind`.